bug: make setup fails due to invalid uv install command in Makefile

### Is there an existing issue for this?

- [x] I have searched the existing issues, it is original

### Description of the bug

**Description**
Running make setup fails at the install-hooks target because it calls uv install, which is not a valid uv subcommand.

**Suggested fix**
Remove the `uv install` line from install-hooks — dependencies are already installed by `uv sync` in the `install-deps` target.

### Steps To Reproduce

**Steps to reproduce**
1. Clone the repository
2. Run `make setup`

**Expected behavior**
Setup completes successfully.

Actual behavior
```
uv install
error: unrecognized subcommand 'install'

  tip: a similar subcommand exists: 'uv pip install'

make: *** [Makefile:22: install-hooks] Error 2
```

### Versions

25.2.1

### Additional Context

<img width="892" height="371" alt="Image" src="https://github.com/user-attachments/assets/250ed4cb-e64c-4e7c-bc60-7d0a4661e65d" />

### Terms and Conditions

- [x] I agree to follow this project's Code of Conduct.

--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,6 @@ install-deps:
 	uv sync --active --all-extras
 
 install-hooks:
-	uv install
 	uv run pre-commit install --install-hooks
 
 run-hooks:

